<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color:white;
}

#centralwidget{
background-color:white;
}


#page {
background-color:white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>8</y>
      <width>782</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#headerLine1 {
color: #10576D;
}

#headerLine2{
color:#10576D;
}

</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="inside_experiment"/>
    <widget class="QWidget" name="outside_experiment">
     <widget class="Line" name="headerLine">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>781</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>10</horstretch>
        <verstretch>10</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#headerLine {
color: #10576D;
}</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="experimentButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/lighExperiment.png</normaloff>
        <normalon>:/darkExperiment.png</normalon>:/lighExperiment.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="historyButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>121</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#historyButton{
	background-color:white; 
	border-bottom:5px solid rgb(227, 227, 227);
	border-right:5px solid rgb(227, 227, 227);
	border-top:5px solid rgb(227, 227, 227);
	border-left:5px solid rgb(227, 227, 227);
	border-radius:40px;
}

#historyButton:pressed {
	background-color:rgb(229, 229, 229);
	border-bottom:5px solid rgb(227, 227, 227);
	border-right:5px solid rgb(227, 227, 227);
	border-top:1px solid rgb(227, 227, 227);
	border-left:1px solid rgb(227, 227, 227);
	border-radius:40px;
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images/resources.qrc">
        <normaloff>:/lightHistory.png</normaloff>:/lightHistory.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="after_experiment"/>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images/resources.qrc"/>
 </resources>
 <connections/>
</ui>
